ngraph/python/tests/test_ngraph/test_create_op.py

```diff
@@ -49,8 +47,8 @@ def test_binary_convolution(dtype):
     input1_shape = [1, 1, 3, 3]
     expected_shape = [1, 1, 7, 7]
 
-    parameter_input0 = ng.parameter(input0_shape, name="Input0", dtype=dtype)
-    parameter_input1 = ng.parameter(input1_shape, name="Input1", dtype=dtype)
+    parameter_input0 = ng.parameter(input0_shape, name="Input0", dtype=np.float32)
+    parameter_input1 = ng.parameter(input1_shape, name="Input1", dtype=ng.utils.types.u1)
```

Comment:
@postrational what do you think about this name?

Reply:
I've added import to ngraph module `__init__` file so now it's avaible as `ng.packed_u1`.